shared/src/main/java/com/faforever/neroxis/util/serial/biome/TerrainMaterials.java

```diff
@@ -1,15 +1,19 @@
 package com.faforever.neroxis.util.serial.biome;
 
 import com.dslplatform.json.CompiledJson;
+import com.faforever.neroxis.map.CubeMap;
 import lombok.Data;
 
+import java.util.List;
+
 @Data
 @CompiledJson
 public class TerrainMaterials {
     // engine limitations - must stay 9 and 10 always
     public static final int TERRAIN_TEXTURE_COUNT = 10;
     public static final int TERRAIN_NORMAL_COUNT = 9;
     private String name;
+    private List<CubeMap> cubeMaps;
```

Comment:
This should be initialized otherwise the importer will throw a null pointer exception